reflex/utils/format.py

```diff
@@ -187,6 +187,21 @@ def to_kebab_case(text: str) -> str:
     return to_snake_case(text).replace("_", "-")
 
 
+def make_default_page_title(app_name: str, route: str) -> str:
+    """Make a default page title from a route.
+
+    Args:
+        app_name: The name of the app owning the page.
+        route: The route to make the title from.
+
+    Returns:
+        The default page title.
+    """
+    title = constants.DefaultPage.TITLE.format(app_name, route)
+    title = title.replace("_", " ").replace("-", " ")
```

Comment:
Can you use `format.to_title_case` here?

Reply:
`format.to_title_case` remove the `_` and `-`, but here I want to replace them with spaces.